runtime/compilers/rillv1/parse_metrics_view.go

```diff
@@ -59,6 +59,16 @@ type MetricsViewYAML struct {
 			Condition string `yaml:"if"`
 		}
 	}
+	DefaultComparison *struct {
```

Comment:
Remove the pointer, just check the values for `!= ""` (reduces chance of nil pointer errors, we've had those for YAML parsing edge cases before)